app/settings_strings.grdp

```diff
@@ -1907,13 +1907,13 @@
     Turn off
   </message>
   <message name="IDS_SETTINGS_ENABLE_DO_NOT_TRACK" desc="The label of the checkbox to enable/disable sending the 'Do Not track' header">
-    Send a "Do not track" request with your browsing traffic
+    Send a "Do Not Track" request with your browsing traffic
```

Comment:
This was recently updated by https://github.com/brave/brave-core/pull/13515

This PR seems to revert that. Did you have an older version of HEAD (which didn't have https://github.com/brave/brave-core/commit/efe80471a924e7bcdd34eba6e9b5324a2266c202)?